site/composer.json

```diff
@@ -52,7 +52,7 @@
   },
   "scripts": {
     "test": "phpunit",
-    "lint": "phpcs --extensions=php ./app",
-    "static-analysis": "phpstan analyze app public/index.php socket/index.php"
+    "lint": "php vendor/bin/phpcs --extensions=php ./app",
+    "static-analysis": "php vendor/bin/phpstan analyze app public/index.php socket/index.php"
```

Comment:
Should these changes be reverted in favor of the changes in https://github.com/Submitty/Submitty/pull/9902?

Reply:
As I mentioned in #9902, this is, in fact, the correct way. I admit I got somewhat unorganized. I will close that PR.

Reply:
Sounds good to me.  You can still remove the files to be analyzed since https://github.com/Submitty/Submitty/pull/9900 has been merged though.

Reply:
done!